src/hooks/useDisableInputUntilReply.ts

```diff
@@ -38,12 +47,7 @@ export const useDisableInputUntilReply = ({
         setIsMessageInputDisabled(false);
       }
     }
-    return () => {
-      if (timerRef.current) {
-        clearTimeout(timerRef.current);
-      }
-    };
-  }, [currentUserId, botUser, lastMessage]);
```

Comment:
I had to remove this because this is being called upon receiving a first streaming bot message. Such that timer is cleared when it should not be.